README.md

```diff
@@ -113,44 +116,77 @@ The blueprint supports `ember-cli-update` to update your addon with any changes
+- Make sure that the chosen addon and test-app locations are all covered by the configured workspace layout of your package manager!
+- Each package should have a distinct name, so make provide unique names for your test apps instead of the default `test-app` by using the `--test-app-name` option.
+- There is no `start` script at the root `package.json` anymore to start both the addon's build and the test app in watch mode. So you would have to run that `start` script with your package manager in both locations in parallel (separate terminal windows/tabs).
+- Pass the `skip-git` option to not auto-commit the generated files. Most likely there will be things to adapt to you specific requirements before committing.
+- The blueprint will omit all files usually generated at the root folder, including `.prettierrc.js`, and instead use whatever you have already defined in your existing monorepo. So you should run the `lint:fix` script for both the addon and the test-app, and eventually address any non-fixable linting issues or other configuration conventions related to your specific setup.
+
+Some examples...
+
+#### Group by name
+
+We group by the name of the addon, the addon's package and its test app are co-located sub-folders:
+
+```
+project-monorepo
+└── addons
+    ├── my-addon
+    │   ├── package
+    │   └── test-app
+    └── ...
+```
+
+[//]: # 'to edit this: https://tree.nathanfriend.io/?s=(%27options!(%27fancy!true~fullPath3~trailingSlash3~rootDot3)~4(%274%27project-monorepo02addons05my-addon0*5package0*5test-app05...%27)~version!%271%27)*%20%200%5Cn5-%203!false4source!5*2%0154320*'
+
+To generate this run:
 
-For example:
 ```bash
-ember addon my-addon-name -b @embroider/addon-blueprint \
+cd project-monorepo
+ember addon my-addon -b @embroider/addon-blueprint \
   --skip-git \
   --skip-npm \
-  --addon-location="package" \
-  --test-app-name="test-app-for-my-addon-name" \
-  --test-app-location="test-app"
-# generates
-#   my-addon-name/
-#     package/
-#     test-app/
+  --addon-location="addons/my-addon/package" \
+  --test-app-name="test-app-for-my-addon" \
+  --test-app-location="addons/my-addon/test-app"
 ```
 
-Then, your workspace search globs can be defined as (for example):
-```js
-# all ember apps in the top level "apps directory"
-apps/*
-# all ember v2 addons share the same structure
-# example:
-#   addons/
-#     my-awesome-addon/
-#       package/
-#       test-app/
-#       docs/
-#       etc/
-addons/*/package
-addons/*/test-app
-addons/*/docs
+#### Group by type
+
+Addons and test-apps are separated:
+
 ```
+project-monorepo
+├── addons
+│   ├── my-addon
+│   └── ...
+└── tests
+    ├── my-addon
+    └── ...
+```
+
+[//]: # 'to edit this: https://tree.nathanfriend.io/?s=(%27options!(%27fancy!true~fullPath2~trailingSlash2~rootDot2)~5(%275%27project-monorepo04738904test39%27)~version!%271%27)*0640862!false3s*my-74-%205source!6%20%207addon8%5Cn9*...%01987654320*'
```

Comment:
I've never seen this tool before. neat!

Reply:
I also just found out about these things today:
* that `[//]: # "..."` is a weird way to write hidden comments in markdown, which I found here: https://stackoverflow.com/questions/4823468/comments-in-markdown
* that web app was a lucky Google hit, as I didn't have the patience to draw that ASCII art by hand 😅